synapse/res/templates/_base.html

```diff
@@ -12,15 +12,7 @@
 </head>
 <body>
 <header class="mx_Header">
-    {% if app_name == "Riot" %}
-        <img src="https://riot.im/img/external/riot-logo-email.png" width="83" height="83" alt="[Riot]"/>
-    {% elif app_name == "Vector" %}
-        <img src="https://matrix.org/img/vector-logo-email.png" width="64" height="83" alt="[Vector]"/>
-    {% elif app_name == "Element" %}
-        <img src="https://static.element.io/images/email-logo.png" width="83" height="83" alt="[Element]"/>
-    {% else %}
-        <img src="https://matrix.org/img/matrix-120x51.png" width="120" height="51" alt="[matrix]"/>
-    {% endif %}
+    <img src="https://beeper.github.io/static/brand/email-header.20240320.2056.png" width="256" height="56" alt="[Beeper]"/>
```

Comment:
do we have a higher res version of this? on retina displays this will show up pixelated/1x

Reply:
We can get one, but we also use this image when sending rageshake emails via the chatwoot API, and we can't include HTML there so we have to control the size of the image by the literal resolution of the image.

I would have to go look and remember how I even got these assets in the first place and then get an upscaled version for this, and then use it when we have the ability to specify HTML.

Reply:
yeah i think the simple fix here is to always use a 2x version and specify the width/height = 1/2 of 2x